CHANGELOG.md

```diff
@@ -0,0 +1,77 @@
+# go-filecoin changelog
+
+## go-filecoin 0.1.4
+
+We're happy to announce go-filecoin 0.1.4. This release contains a better install experience, Proof-of-Spacetime integration, more reliable message sending and networking, and many other improvements. Get pumped! 🎁
+
+### Install and Setup
+
+#### 💝 Binary releases
+
+Linux and MacOS binaries for go-filecoin are now available! See [Installing from binary](https://github.com/filecoin-project/go-filecoin/wiki/Getting-Started#installing-from-binary) for instructions.
+
+#### 🍱 Precompiled proofs parameters
+
+Running secure proofs requires parameter files that are several GB in size. Previously, these files were generated at install, an extremely memory-intensive process causing slow or impossible builds for many users. Now, you can download pre-generated files during install by running `paramfetch`. This step is now included in the [Installing from binary](https://github.com/filecoin-project/go-filecoin/wiki/Getting-Started#installing-from-binary) instructions.
+
+#### 🦖 Version checking
+
+go-filecoin now checks that it is running the same version (at the same commit) before connecting to a devnet. This is a temporary fix until a model for change is implemented, allowing different versions to interoperate.
+
+### Features
+
+#### 💎 Proof-of-Spacetime Integration
+
+For each storage deal, miners now call `rust-fil-proofs` to periodically generate proofs of continued storage. With this major integration in place, you can expect some follow-up  (for example, storage mining faults do not yet appear on-chain) and continued optimizations to the underlying Proof-of-Spacetime implementation.
```

Comment:
should probably say its not the final construction

Reply:
done